roles/openvpn/README.md

```diff
@@ -0,0 +1,137 @@
+# [openvpn](#openvpn)
```

Comment:
This is OK for now, but we'll need to re-write it so it's consistent for the other ce-provision docs.